mono/mini/aot-runtime.c

```diff
@@ -5373,6 +5375,11 @@ load_function_full (MonoAotModule *amodule, const char *name, MonoTrampInfo **ou
 			if (ji->type == MONO_PATCH_INFO_SPECIFIC_TRAMPOLINE_LAZY_FETCH_ADDR) {
```

Comment:
Should probably be a switch. I was going to do that months ago but limited the churn (when I was removing nearby hashtable use).